src/main/charts/confluence/values.yaml

```diff
@@ -153,6 +153,10 @@ confluence:
   # See https://hub.docker.com/r/atlassian/confluence-server for supported variables.
   additionalEnvironmentVariables: []
 
+  jvmDebug:
+    # -- If set to true, Confluence JVM will be started with debugging port open.
```

Comment:
For clarity can we include the port number (5005) in this comment

Reply:
Added